voluncheer/opportunityboard/test_models.py

```diff
@@ -3,13 +3,13 @@
 from profiles.models import Organization
 from profiles.models import User
 from profiles.models import UserType
-from jobboard.models import Job
+from opportunityboard.models import Opportunity
```

Comment:
Keep sorted alphabetically. I'm really surprised our linter doesn't require this.

```suggestion
from opportunityboard.models import Opportunity
from profiles.models import Organization
from profiles.models import User
from profiles.models import UserType
```